lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -179,14 +179,17 @@ class UnusedCSSRules extends Audit {
       return UnusedCSSRules.mapSheetToResult(indexedSheets[sheetId], pageUrl);
     }).filter(sheet => sheet && sheet.wastedBytes > 1024);
 
+
+    const headings = [
+      {key: 'url', itemType: 'url', text: 'URL'},
```

Comment:
in all the css-based ones the "url column" doesn't always have a url and it's a code snippet preview instead since it's an inline style. it'd be nice to have the details renderer support that too so the code snippets can say `{type: "code", content: ".inline-style {...}"}` or something